src/CommandingOfficers/CommanderAve.java

```diff
@@ -344,7 +330,23 @@ private void relevelSnow(GameMap gameMap, GameEventQueue outEvents)
     {
       int oldVal = snowMap[dis.xCoord][dis.yCoord];
       snowMap[dis.xCoord][dis.yCoord] = (oldVal - SNOW_MELT_RATE < 0)? 0 : oldVal - SNOW_MELT_RATE;
-      log("Snow at " + dis + " melting from " + oldVal + " to " + snowMap[dis.xCoord][dis.yCoord]);
+      if( snowLoggingEnabled ) log("Snow at " + dis + " melting from " + oldVal + " to " + snowMap[dis.xCoord][dis.yCoord]);
+    }
+
+    // Update weather forecast.
+    ArrayList<MapChangeEvent.EnvironmentAssignment> tiles = new ArrayList<MapChangeEvent.EnvironmentAssignment>();
+    for (XYCoord coord : toSnow)
+    {
+      Environment newEnvi = Environment.getTile(gameMap.getEnvironment(coord).terrainType, Weathers.SNOW);
+      int duration = snowMap[coord.xCoord][coord.yCoord] / SNOW_MELT_RATE;
```

Comment:
This should probably still have the +1; it looks a little weird for Ave's snow to disappear and reappear every turn, heh.

Reply:
Hmm. There is a bit of a flicker for tiles that aren't 200+ deep. On the other hand, adding a spurious extra 1 to duration causes them to stay under snow longer than they should once they become disjointed.

Reply:
Hmm.
I don't think it should make a *huge* difference in gameplay either way; an extra turn of snow after Ave loses a property probably isn't a huge deal.
I guess it makes a lot less difference with only a frame or two of flicker. Merge if you like, I guess.

Reply:
Eventually I'd like to support zero-animation game advancement. I'll leave the flicker for now because a) it's technically more correct, and b) so it can be auto-fixed once we get zero-animation working.